package.json

```diff
@@ -39,7 +39,7 @@
     "five-bells-shared": "^21.2.0",
     "ilp-core": "~11.1.0",
```

Comment:
Note that this needs to be upgraded to https://github.com/interledgerjs/ilp-core/pull/79 before merging.